tasks/changelog.js

```diff
@@ -0,0 +1,93 @@
+const fs = require(`fs`)
+const { promisify } = require(`util`)
+const { join } = require(`path`)
+const exec = promisify(require(`child_process`).exec)
+const inquirer = require(`inquirer`)
+
+const changes = []
+
+const ask = async () => {
+  const answer = await inquirer
+    .prompt([
+      {
+        type: `list`,
+        name: `type`,
+        message: `What type of change do you want to add to the changelog?`,
+        choices: [
+          { name: `Addition of feature`,
+            value: `Added` },
+          { name: `Change of existing behavior`,
+            value: `Changed` },
+          { name: `Fix for a bug`,
+            value: `Fixed` },
+          { name: `Security improvement`,
+            value: `Security` },
+          { name: `Deprecation of unused code/feature`,
+            value: `Deprecated` }
+        ]
+      },
+      {
+        type: `input`,
+        name: `content`,
+        message: `What is the content of the change?`,
+        validate: function(value) {
+          if (value) return true
+
+          return `You need to specify the change.`
+        }
+      },
+      {
+        type: `input`,
+        name: `issue`,
+        message: `(optional) Which GitHub issue is discussing this change?`
```

Comment:
can we make this mandatory ? adding the PR number also works

Reply:
Sure we can. Can you tell what the benefits are of adding the PR number to the changelog?

Reply:
so people can follow the link to the PR that introduced a specific entry (it can be useful for reference, ease of debugging, etc)

Reply:
Nice idea, it gives context in the long run if someone wants to come to the repo and read what happened / why stuff was done

Reply:
Or we can put it on a blockchain :D

Reply:
okidoki, doing it

Reply:
done